public/stylesheets/site/2.0/10-types-groups.css

```diff
@@ -171,7 +171,7 @@ li.relationships a {
   float: none;
 }
 
-.kudos {
+p.kudos {
```

Comment:
Add the p selector so this only targets the kudos block on works themselves and not the kudos dd in stats.